src/others/components/map/Map.tsx

```diff
@@ -30,12 +30,21 @@ export const Map = ({ sourceWithLayer, interactiveLayerIds }: MapProps) => {
   const { t } = useTranslation();
   const mapRef = useRef<MapRef>(null);
   const [popupInfo, setPopupInfo] = useState<PopupInfo | null>(null);
+  const [hoveredRegionId, setHoveredRegionId] = useState<number | string | undefined>(undefined);
   const [cursor, setCursor] = useState<"auto" | "pointer">("auto");
 
   const closePopup = useCallback(() => {
     setCursor("auto");
     setPopupInfo(null);
-  }, []);
+    if (mapRef?.current) {
```

Comment:
[nit] In https://github.com/ukraine-taskforce/requests-dashboard/pull/71, I wrapped the page in `MapProvider` so we have access to the [`useMap()`](https://visgl.github.io/react-map-gl/docs/api-reference/use-map) hook, so we don't need to reference the `ref` anymore.

Reply:
done